static/src/stylesheets/module/experiments/_remind-me.scss

```diff
@@ -0,0 +1,41 @@
+$remind-me-color: $features-support-1;
+.remind-me {
+    background-color: $remind-me-color;
+    padding: $gs-baseline/4 $gs-gutter/2 $gs-baseline/2;
+    border-top: 3px solid $features-support-3;
+    color: #ffffff;
+}
+
+.remind-me__subtitle {
+    @include fs-textSans(5);
+    font-weight: bold;
```

Comment:
Yeah, subtitle might not be the best name but couldn't think of a better one... "surtitle" maybe??